.gitignore

```diff
@@ -7,4 +7,6 @@ build/
 
 node_config.json
 .priv
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+
+.DS_Store
```

Comment:
`.DS_Store` — better put such stuff to global .gitignore (it’s related to developer’s env, not project)